posthog/queries/funnels/base.py

```diff
@@ -426,29 +423,27 @@ def _get_inner_event_query(
             # where i is the starting step for exclusion on that entity
             all_step_cols.extend(step_cols)
 
-        steps = ", ".join(all_step_cols)
-
         breakdown_select_prop = self._get_breakdown_select_prop()
-        if len(breakdown_select_prop) > 0:
-            select_prop = f", {breakdown_select_prop}"
-        else:
-            select_prop = ""
+
+        if breakdown_select_prop:
+            all_step_cols.append(breakdown_select_prop)
+
         extra_join = ""
 
         if self._filter.breakdown:
             if self._filter.breakdown_type == "cohort":
                 extra_join = self._get_cohort_breakdown_join()
+
             else:
                 values = self._get_breakdown_conditions()
                 self.params.update({"breakdown_values": values})
 
-        inner_query = FUNNEL_INNER_EVENT_STEPS_QUERY.format(
-            steps=steps,
-            event_query=event_query,
+        extra_select_fields = f", {', '.join(all_step_cols)}" if all_step_cols else ""
+
+        inner_query = inner_query.format(
```

Comment:
Q: Should we name this `events_query` or `funnel_events_query`? inner_query had me going in cycles since it was ambigious.